wicket-core/src/main/java/org/apache/wicket/markup/html/list/ListView.java

```diff
@@ -652,4 +653,24 @@ public final void setModelObject(List<T> object)
 	{
 		setDefaultModelObject(object);
 	}
+
+	/**
+	 * Returns a new ListVIew
+	 * @param id component id
+	 * @param model model containing a list of items
+	 * @param consumer lambda to use for populating the ListView
+	 * @param <T> type of the items in the model
+	 * @return a new ListView
+	 */
+	public static <T> ListView<T> populateItem(final String id, final IModel<? extends List<T>> model, final SerializableConsumer<ListItem<T>> consumer) {
```

Comment:
For some reason I don't like the name of the method. It doesn't fit well for this factory method. But I don't have a better one at the moment.
Also it should be added to org.apache.wicket.lambda.Lambdas